addons/binding/org.openhab.binding.sinope/readme.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: documentation
+---
+
+{% include base.html %}
+
+# Binding for Sinopé devices
+
+This binding supports [Eclipse SmartHome API] (<https://www.eclipse.org/smarthome/index.html>).
+The integration happens through the Sinopé (GT150) bridge, which acts as an IP gateway to the Sinopé devices on the 916 Mhz ISM band.
+
+This binding supports multiple gateways with multiple devices.
+
+---
+
+## Supported Things
+
+The Sinopé bridge is required as a "bridge" for accessing any other Sinopé devices.
+
+Right now, only the thermostat devices (3000W and 4000W) (TH1120RF) are supported.
+
+## Discovery
+
+The Sinopé Gateway (bridge) discovery is not supported for now. It will be added in future release.
+The Sinopé devices discovery is implemented. 
+
+## Prerequisites
+
+### Bridge or the Sinopé Gateway 
+First, you will need to get your API key from your Sinopé gateway.
+
+Grab the latest release of the [sinope-core library](<https://github.com/chaton78/sinope-core/releases>)
```

Comment:
I can only find pre-releases there. And in any case: I'd think that this is rather nasty for many non-tech users and requiring additional software just for the configuration is far from being ideal.

As the sinope-core library is included in the binding I would expect that all required code is already there.
So why can't you simply take the gateway ID as a configuration parameter and request the API key by the handler?